### DISCUSSION TITLE
天竺鼠車車_芝麻日常

### DISCUSSION THREAD

**Discussion by Lipunpun** [1101-GS4719-C/Python-Community-GS4719-C#1193] [General]
![image](https://user-images.githubusercontent.com/125544059/229037963-8c470aa8-b877-43b9-9299-7a78fa9173bb.jpeg)

芝麻是一隻很安靜的小女生，之前也有養過一隻公的（叫做kimchi)，喜歡一直叫就算了，他還是公的，成鼠後那個味道真的很恐怖，所以芝麻還算是很好的！

平常時，我覺得他的比較像是貓咪，不理他的時候就會一直看著我，不然就是在睡覺，但如果要去找他玩，他會開心到一直原地轉圈圈、爆米花跳，缺點是他有點膽小如果是陌生人看到他他就不敢出來了。

總之，天竺鼠還是很不錯的寵物，只是因為每一隻個性都不一樣，要有心理準備他會拆家、隨地大便喔！

**Comment by Lipunpun** [comment]
天竺鼠的壽命大概五到六年，飼養前我希望大家可以想清楚再買，後續的醫療、美容、飼料都是一筆不小的支出，我現在在他身上花的說不定跟養狗養貓有得比，大家要想清楚不要讓天天寂寞喔。![image](https://user-images.githubusercontent.com/125544059/229055373-b98bc42a-3340-4f71-a135-27159b53625d.jpeg)

**Comment by pigeon9975** [comment]
天竺鼠真的超級可愛！！可惜我家沒有足夠的空間養，再加上現在都住宿舍，一周只有不到兩天的時間可以在家陪他，看來只可遠觀不可褻玩阿QQ
謝謝你分享可愛的寵物，讓我可以在討論群上看到好療癒的照片！
